object_storage_api/services/image.py

```diff
@@ -75,6 +75,25 @@ def create(self, image_metadata: ImagePostMetadataSchema, upload_file: UploadFil
 
         return ImageSchema(**image_out.model_dump())
 
+    def get(self, image_id: str) -> ImageGetUrlInfoSchema:
+        """
+        Retrieve an image with its presigned url by its ID.
+        :param: ID of the image to retrieve.
+        :return: An image or None if no image is retrieved.
```

Comment:
```suggestion
        Retrieve an image with its presigned url by its ID.
        
        :param: ID of the image to retrieve.
        :return: An image or `None` if no image is retrieved.
```
You should also add `:raises` here like in the create method above. Doesn't return `None` anymore.